pinot-common/src/main/java/com/linkedin/pinot/common/metadata/segment/IndexLoadingConfigMetadata.java

```diff
@@ -32,10 +32,14 @@
   public static final String KEY_OF_LOADING_INVERTED_INDEX = "metadata.loading.inverted.index.columns";
   public static final String KEY_OF_SEGMENT_FORMAT_VERSION = "segment.format.version";
   public static final String KEY_OF_ENABLE_DEFAULT_COLUMNS = "enable.default.columns";
+  public static final String KEY_OF_STAR_TREE_FORMAT_VERSION = "startree.format.version";
+
   private final Set<String> _loadingInvertedIndexColumnSet = new HashSet<String>();
   private final String DEFAULT_SEGMENT_FORMAT = "v1";
```

Comment:
I intend to do a global renaming of StarTreeV1/V2, and it will span many files outside of this PR. Will file a separate PR for that.